lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -252,7 +252,7 @@ protected void PeriodicUploadLog(object source, ElapsedEventArgs e)
             }
         }
 
-        internal string[] ReadLogFile(string logFileName, string subfolder = "driver", string testFolder = DefaultRuntimeFolder, int retryCount = 60)
+        internal string[] ReadLogFile(string logFileName, string subfolder = "driver", string testFolder = DefaultRuntimeFolder, int retryCount = 100)
```

Comment:
broadcast tests usually takes longer time for the log files to be completed. I will  change it by passing parameters.